src/core/config/config.go

```diff
@@ -231,7 +231,7 @@ func RegistryURL() (string, error) {
 
 // InternalJobServiceURL returns jobservice URL for internal communication between Harbor containers
 func InternalJobServiceURL() string {
-	return strings.TrimSuffix(cfgMgr.Get(common.JobServiceURL).GetString(), "/")
+	return os.Getenv("JOBSERVICE_URL")
```

Comment:
why not continue using common.JobServiceURL

Reply:
The original implementation gets the URL from config manager which reads it from the environment variable`JOBSERVICE_URL` as well. The config manager depends on database when starting up, this is unnecessary for jobservice URL.